vtv-shows-app/src/components/features/review/ReviewForm.tsx

```diff
@@ -0,0 +1,58 @@
+// src/components/features/shows/ReviewForm.tsx
+import { useState } from "react";
+import { Box, Button, Input, Textarea } from "@chakra-ui/react";
+
+interface Review {
+  email: string;
+  avatarUrl?: string;
+  rating: number;
+  comment: string;
+}
+
+interface ReviewFormProps {
+  addShowReview: (review: Review) => void;
+}
+
+const ReviewForm = ({ addShowReview } : ReviewFormProps) => {
+  const [email, setEmail] = useState("");
+  const [rating, setRating] = useState(0);
+  const [comment, setComment] = useState("");
+
+  const handleSubmit = () => {
+    const newReview = { email, rating, comment };
+    addShowReview(newReview);
+    setEmail("");
+    setRating(0);
+    setComment("");
+  };
+
+  return (
+    <Box>
+      <Input
+        placeholder="Your email"
+        value={email}
+        onChange={(e) => setEmail(e.target.value)}
+        mb="2"
+      />
+      <Input
+        placeholder="Rating (0-5)"
```

Comment:
I can still type in 1234315 in this input and be allowed to post the review. There is no validation on it. Same goes for all other input fields in the form. We would need some kind of input validation to prevent errors.